shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/viewModel/FavoritesViewModelTest.kt

```diff
@@ -0,0 +1,315 @@
+                                    setOf(stop2.id),
+                                    upcomingTrips =
+                                        listOf(
+                                            objects.upcomingTrip(
+                                                predictions
+                                                    .getValue(stop2)
+                                                    .getValue(route2)
+                                                    .getValue(1)
+                                            )
+                                        ),
+                                    alertsHere = emptyList(),
+                                    allDataLoaded = true,
+                                    hasSchedulesToday = false,
+                                    alertsDownstream = emptyList(),
+                                    context = RouteCardData.Context.Favorites,
+                                )
+                            ),
+                            globalData,
+                        )
+                    ),
+                    now,
+                ),
+            )
+        fun FavoritesViewModel.State.tweakStaticDataTime() =
+            this.copy(
+                staticRouteCardData =
+                    this.staticRouteCardData?.map { routeCardData ->
+                        routeCardData.copy(at = staticDataTime)
+                    }
+            )
+
+        testViewModelFlow(viewModel).test {
+            assertEquals(
+                FavoritesViewModel.State(
+                    favorites = null,
+                    isReturningFromBackground = false,
+                    routeCardData = null,
+                    staticRouteCardData = null,
+                ),
+                awaitItem(),
+            )
+            assertEquals(
+                FavoritesViewModel.State(
+                    favorites = favorites.routeStopDirection,
+                    isReturningFromBackground = false,
+                    routeCardData = null,
+                    staticRouteCardData = null,
+                ),
+                awaitItem(),
+            )
+            assertEquals(
+                FavoritesViewModel.State(
+                    favorites = favorites.routeStopDirection,
+                    isReturningFromBackground = false,
+                    routeCardData = null,
+                    staticRouteCardData = expectedStaticData,
+                ),
+                awaitItem().tweakStaticDataTime(),
+            )
+            assertEquals(
+                FavoritesViewModel.State(
+                    favorites = favorites.routeStopDirection,
+                    isReturningFromBackground = false,
+                    routeCardData = expectedRealtimeData,
+                    staticRouteCardData = expectedStaticData,
+                ),
+                awaitItem().tweakStaticDataTime(),
+            )
+        }
+    }
```

Comment:
suggestion: 
I think it would be worthwhile to add some basic tests for the fireEvent wrappers too.